process/smartContract/process.go

```diff
@@ -927,30 +914,37 @@ func (sc *scProcessor) createSmartContractResult(
 	return result
 }
 
+func (sc *scProcessor) computeActualConsumedGas(
+	tx data.TransactionHandler,
+	vmOutput *vmcommon.VMOutput,
+	acntSnd state.UserAccountHandler,
+) uint64 {
+	consumedGas := tx.GetGasLimit() - vmOutput.GasRemaining
+	if check.IfNil(acntSnd) {
+		consumedGas = consumedGas - sc.economicsFee.ComputeGasLimit(tx)
```

Comment:
```GO
consumedGas -= sc.economicsFee.ComputeGasLimit(tx)
```

as to be consistent with the rest of the function

Reply:
done